modules/lib/api/document.xql

```diff
@@ -2,8 +2,8 @@ xquery version "3.1";
 
 module namespace dapi="http://teipublisher.com/api/documents";
 
-import module namespace router="http://exist-db.org/xquery/router";
-import module namespace errors = "http://exist-db.org/xquery/router/errors";
+import module namespace router="http://e-editiones.org/roaster";
```

Comment:
please import as roaster to avoid confusion with the router lib

Reply:
see my earlier comment, that was a display glitch which has now been rectified somehow

Reply:
I still see `import module namespace router=` instead of `roaster=`